addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -10,13 +10,18 @@ Currently, the binding supports a single type of Thing, being the ```command```
 
 The binding does not require any specific configuration.
 
-Note that the commands are executed in the context and with the privileges of the process running the java virtual machine.
-It is not advised to run the virtual machine as superuser/root.
+
+Note that the commands are executed in the context and with the privileges of the process running the java virtual machine. It is not advised to run the virtual machine as superuser/root. Linux Os needs the user openhab/openhabian to be able to execute dedicated command, it is advised to always test in the command line if this is possible.
```

Comment:
I believe it's more common to capitalize  Java Virtual Machine. Also I wonder if you'd like to add a few words of explanation like: "in which openHAB is executed."

I'd make the new sentence a new paragraph and write:

```text
**Linux:**
On a Linux system the system user `openhab` needs to have the privileges needed to execute your intended  command.
It is advised to test the correct operation of the command in the scope of the `openhab` user by on the command line: